src/doc/book/no-stdlib.md

```diff
@@ -87,12 +93,18 @@ pub extern fn main(_argc: i32, _argv: *const *const u8) -> i32 {
     0
 }
 
-// These functions and traits are used by the compiler, but not
+// These functions are used by the compiler, but not
 // for a bare-bones hello world. These are normally
 // provided by libstd.
 #[lang = "eh_personality"]
 #[no_mangle]
-pub extern fn eh_personality() {
+pub extern fn rust_eh_personality() {
+}
+
+// This function may be needed based on the compilation target.
+#[lang = "eh_unwind_resume"]
+#[no_mangle]
```

Comment:
`#[no_mangle]` seems questionable, as this is not done in `libpanic_unwind`. The `extern "C"` might already indicate that those functions are not supposed to be mangled. I'm not quite sure on that though, so I kept it for now.